src/odemis/dataio/tiff.py

```diff
@@ -1243,6 +1252,11 @@ def _addImageElement(root, das, ifd, rois, fname=None, fuuid=None):
             for j in range(3):
                 trane.attrib["A%d%d" % (i, j)] = "%.15f" % trans_mat[i][j]
 
+    if model.MD_EXTRA_SETTINGS in globalMD:
+        sett = globalMD[model.MD_EXTRA_SETTINGS]
+        sett = json.dumps(sett)  # serialize hw settings
+        extrase = ET.SubElement(ime, "ExtraSettings", attrib={"Settings": sett})
```

Comment:
Do not use an attribute inside the element, but the *content* of the element, aka `.text`.